mlrun/model_monitoring/writer.py

```diff
@@ -227,10 +232,20 @@ def do(self, event: _RawEvent) -> None:
         _Notifier(event=event, notification_pusher=self._custom_notifier).notify()
 
         if mlrun.mlconf.alerts.mode == mlrun.common.schemas.alert.AlertsModes.enabled:
+            endpoint_record = get_endpoint_record(
```

Comment:
I think it is nice to have the model name in the message, but I think we don't want to access the db in each writer run. Try to save all the model_endpoint you already get from the db like we are doing in the `EventStreamProcessor::ProcessEndpointEvent.resume_state`.